apis/python/tests/test_query_condition.py

```diff
@@ -50,6 +50,7 @@ def soma_query(uri, condition):
         'louvain == "NK cells"',  # string
         "percent_mito > 0.02",  # float
         "is_b_cell == True",  # bool
+        "ord == None",  # None values
```

Comment:
I think it is worth adding all of the other operators that support testing for None/NA, e.g., `ord != None`.  Cheap to test, and may catch edge cases.